src/reuse/lint.py

```diff
@@ -339,6 +270,14 @@ def run(args, project: Project, out=sys.stdout):
     with contextlib.ExitStack() as stack:
         if args.quiet:
             out = stack.enter_context(open(os.devnull, "w", encoding="utf-8"))
```

Comment:
Instead of doing this, we could create a `format_quiet`.

Reply:
This should be resolved now.